lib/vdsm/virt/backup.py

```diff
@@ -365,6 +377,14 @@ def _start_monitoring_scratch_disks(vm, backup_disks, backup):
             }
```

Comment:
I don't know why you need so much debug prints, when the actual change is just:

    "dom_id": backup_disk.scratch_disk.dom_id,
    ...

And updating the to print backup_disk.drive.scratch_disk instead of formatting all
the info in the log.  The log will show the scratch disk in a very clear way, there is no need for all this debug effort.